src/mocks/api/shopsResult.ts

```diff
@@ -0,0 +1,31 @@
+import { http, HttpResponse } from 'msw';
+import { API_PATH } from 'config/constants';
+import shopsResultData from './data/shopsResultData';
+
+interface RequestBody {
+  lat: number | undefined;
+  lng: number | undefined;
+}
+
+const handlers = [
+  http.post(`${API_PATH}/shops`, async ({ request }) => {
+    const body = await request.json() as RequestBody;
```

Comment:
1. mocking 된 api 이기 때문에 앞에 굳어 API_PATH 를 명시해주지 않아도 될 것 같습니다.
2. assertion 보다는 제네릭을 사용해 보면 좋을 것 같네요.

```suggestion
interface PathParams {
  keyword: string;
}

  http.post<PathParams, RequestBody>('/shops', async ({ request }) => {
    const body = await request.json();
```

Reply:
이 부분 경로를 API_PATH로 명시해 주지 않으면 Service Worker랑 서버 api랑 동시에 발생돼서 수정하지 못 했습니다. 혹시 이유가 뭔지 알 수 있을까요...?

Reply:
음... 우선 지금 axios.create 를 사용해서 API_PATH를 base url 로 받고 있는게 문제 같네요.
전용 MSW 환경이 없어서 지금 API_PATH 를 위에서 지워버리게 되면 msw handler에서는 가로채지만, axios.create 로 만들어진 새 요청이 생겨버려요.
msw 환경을 이제 별도로 둘 수 있다고 하면 msw 환경에서는 API_PATH 를 '/' 로 두고 제가 전 코멘트에서 말한것처럼 삭제하고 쓰면 됩니다.

그런데 지금 문제가.. msw 코드가 이것 밖에 없다보니 절대 경로를 명시해줄 수 밖에 없는 것 같네요..
그래서 msw 전용으로 axios create 해 두는것도 좋을 것 같고, 우선 대의님처럼 두어도 절대 경로이긴 하니 괜찮을 것 같아요!
아래 예시도 첨부합니다.

<img width="604" alt="image" src="https://github.com/BCSDLab/JJBAKSA_FRONT_END/assets/51052049/1cfd1317-5e6b-4c77-8d6b-0f2b3bdd74c2">